fiftyone/core/video.py

```diff
@@ -574,6 +578,10 @@ def make_frames_dataset(
         to_sample_view = sample_collection._root_dataset.select(
             ids_to_sample, ordered=True
         )
+
+        if output_dir:
+            output_dir = fou.normalize_path(output_dir)
```

Comment:
Defer any normalization as long as possible? IE push to `sample_videos()` if you're going to include it?